### ISSUE TITLE
Deprecation: Remove nn.functional.sigmoid 

### ISSUE THREAD

**Issue by mruberry** [pytorch/pytorch#65910]
This operation is deprecated and warns whenever it's used, see it's implementation here:

https://github.com/pytorch/pytorch/blob/4666e3f19228ddb73294c3ce7eba01b1641c083a/torch/nn/functional.py#L1847

The deprecation warning also correctly points users to torch.sigmoid, for which nn.functional.sigmoid is an alias. We should finally remove this operator.

**Comment by riotonzuk** [comment]
Hi, I can also clear this up
